src/pages/index.js

```diff
@@ -1,3 +1,29 @@
+import "./index.css";
+import {
+  enableValidation,
+  resetValidation,
+  settings,
+  disableButton,
+} from "../scripts/validation.js";
+import logoSrc from "../images/logo.svg";
+import avatarSrc from "../images/avatar.jpg";
+import plusSrc from "../images/plus.svg";
+import pencilSrc from "../images/pencil.svg";
+import Api from "../utils/Api.js";
+import { setButtontext } from "../utils/helpers.js";
+
+const spotsLogo = document.getElementById("spots-logo");
+spotsLogo.src = logoSrc;
+
+const bessieAvatar = document.getElementById("bessie-avatar");
```

Comment:
`bessieAvatar` should be named `avatarElement` because it's not only for `Bessie`. It's for the custom avatar of the user